make_level.py

```diff
@@ -0,0 +1,217 @@
+            filedata = fp.read()
+
+        # Add import statement
+        replace = "\n".join([
+            "from gitgud.skills.{0} import skill as {0}_skill".format(skill_name),
+            "",
+            "from gitgud.skills.u"
+        ])
+        filedata = filedata.replace("\nfrom gitgud.skills.u", replace)
+
+        # Add to input array of Skill
+        replace = ",\n    {}_skill\n]".format(skill_name)
+        filedata = filedata.replace("\n]", replace)
+
+        # Write to file
+        filepath = os.path.join("gitgud", "skills", "__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Registered skill \"{}\" in {}".format(skill_name, filepath))
+
+        write_init(skill_name, skill_path, level_name)
+        return
+    else:
+        # Populate file with BasicLevel
+        with open(os.path.join(skill_path, "__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        replace = ",\n        BasicLevel('{level_name}', pkg_resources.resource_filename(__name__, '_{level_name}/'))\n    ]".format(level_name=level_name)
+        filedata = filedata.replace("\n    ]", replace)
+        filedata = filedata.replace("[,", "[")
+        filepath = os.path.join(skill_path, "__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Registered level \"{}\" in {}".format(level_name, filepath))
+    return
+
+
+def write_test(skill_name, skill_path, level_name, level_path):
+    if not os.path.exists(os.path.join(skill_path, "test_levels.py")):
+        with open(os.path.join(skill_path, "test_levels.py"), 'w+') as fp:
+            test_setup = "\n".join([
+                "import pytest",
+                "",
+                "from gitgud.skills.testing import simulate",
+                "",
+                "from . import skill",
+                "",
+                "",
+                "level_tests = [",
+                "    (",
+                "        skill[\'{}\'], [".format(level_name),
+                "            \'git gud commit\',  # Example, change to solution for your level",
+                "            \'git merge example\'",
+                "        ]",
+                "    )",
+                "]",
+                "",
+                "",
+                "@pytest.mark.parametrize('level,commands', level_tests)",
+                "def test_level(gg, level, commands):",
+                "    simulate(gg, level, commands)",
+                ""
+            ])
+            fp.write(test_setup)
+            print("Added entry for {}".format(level_path))
+    else:
+        with open(os.path.join(skill_path, "test_levels.py"), 'r') as fp:
+            filedata = fp.read()
+
+        replace = "\n".join([
+            "    ), (",
+            "        skill[\'{}\'], [".format(level_name),
+            "            #Example: \'git gud commit\',",
```

Comment:
Can you change this to match what I did above?

Reply:
done